dask_jobqueue/deploy/cluster_manager.py

```diff
@@ -131,7 +131,10 @@ def adapt(
             elif maximum_memory is not None:
                 kwargs["maximum"] = self._get_nb_workers_from_memory(maximum_memory)
         self._adaptive_options.update(kwargs)
-        self._adaptive = Adaptive(self.scheduler, self, **self._adaptive_options)
+        try:
+            self._adaptive = Adaptive(self.scheduler, self, **self._adaptive_options)
+        except Exception:
+            self._adaptive = Adaptive(self, **self._adaptive_options)
```

Comment:
We dropped the need to explicitly pass in the scheduler in recent versions.  We will query the scheduler over a network connection instead so that it can be remote in the future.

Reply:
That's perfect!